src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -556,6 +572,18 @@ internal bool IsNetStandardFeatureFlagEnabled()
             return headerTags;
         }
 
+        internal string[] ParseUrlStringArray(string commaSeparatedValues)
```

Comment:
This is not removing empty entries like the LINQ snippet did. Empty strings will match all urls and break this feature. For example, `datadog.com,` or `datadog.com,,google.com` will incur the overhead but will not skip anything. 

This method probably deserves a unit test.